hera_cal/io.py

```diff
@@ -641,7 +641,7 @@ def partial_write(self, output_path, data=None, flags=None, nsamples=None,
                                   this.nsample_array, **self.last_read_kwargs)
 
     def iterate_over_bls(self, Nbls=1, bls=None, chunk_by_redundant_group=False, reds=None,
-                         bl_error_tol=1.0, include_autos=True):
+                         bl_error_tol=1.0, include_autos=True, freqs_to_load=None):
```

Comment:
To better match other iterators, I'd call this `freqs` or `frequencies` rather than `freqs_to_load`

Reply:
calling it frequencies.